internal/provider/resources/deployment_test.go

```diff
@@ -100,6 +103,7 @@ func TestAccResource_deployment(t *testing.T) {
 		Version:                "v1.1.1",
 		WorkPoolName:           "evergreen-pool",
 		WorkQueueName:          "evergreen-queue",
+		StorageDocumentID:      "a489d88c-4d5e-44cb-ae57-2a8ebb3b6ae2",
```

Comment:
This is just some demo GitHub repo block I created for testing. We'll want to update this to something reliable, potentially a resource we create in the test.

Reply:
This is currently referring to a `mitch-demo-repo` block.